kong/plugins/jwt/handler.lua

```diff
@@ -28,7 +28,13 @@ local function retrieve_token(request, conf)
     end
   end
 
-  local authorization_header = request.get_headers()["authorization"]
+  local jwt_key = "authorization"
```

Comment:
would it be useful to make this name configurable?

Reply:
thx Tieske,i have adjust it